docs/topics/configmap-mission-deploy-booster.adoc

```diff
@@ -25,7 +25,7 @@ $ oc new-project {project-name}
 ----
 
 ifdef::configmap-vertx,configmap-spring-boot-tomcat[]
-. Be sure that view access rights for the service account are added before deploying your booster.
+. Assing view access rights to the service account before deploying your booster, to allow the OpenShift API to access the contents of the ConfigMap.
```

Comment:
"Assing" -> "Assign".

Also "to allow the OpenShift API to access the contents of the ConfigMap" is backwards. It is to allow the service account to access the OpenShift API.

I'd reformulate like this:

> Assign the necessary permissions to the service account before deploying your booster, so that the booster can access the OpenShift API in order to read the contents of the ConfigMap.